installer/scripts/service_control

```diff
@@ -93,6 +102,7 @@ start_omsagent_process()
 
 start_omsagent()
 {
+    exit_if_agent_not_onboarded
```

Comment:
Yes. Start is a subset of enable, but enable should JUST enable and not start.

By the same token, disable is a superset of stop (or, said another way, stop is a subset of disable).

It is incorrect for enable to start the service. Enable should simply perform service registration, but not start the service.